docs/content.md

```diff
@@ -694,3 +508,48 @@ A Replay in which you travelled around and discovered a lot of terrain is signif
 
 An average Replay File of **10 Minutes duration** usually is between **2MB and 10MB large**.
 Replays recorded on **Minigame Servers** with lots of particle effects and world changes might be larger.
+
+# Troubleshooting [troubleshooting]
+> If you have trouble with rendering, please first consult the Documentation before asking for help in the Discord.
+
+## No FFmpeg installation found [ffmpeg]
+![](img/ffmpeg-missing.jpg)
+The error screen that is displayed when no FFmpeg installation could be found
+
+If you have not installed ffmpeg, please follow the steps provided [here](#installing-ffmpeg)
+
+If already have installed ffmpeg, you have to manually tell the **Replay Mod** where your FFmpeg executable is located.
+
+First, get the full path to your `ffmpeg.exe` (on Windows) or `ffmpeg` executable (on Mac/Linux).  
+On Windows, this path  might look like `C:/ffmpeg/ffmpeg.exe`.
+
+Then, open the **"Command Line Settings" Tab** in the Render Settings and paste this path into the **left input field** and retry rendering.
+
+## Crash while rendering [crash]
+If Minecraft crashes after a few frames of rendering, it most likely means the **FFmpeg** didn't like the **Command Line Arguments** you passed.
+If you customized the Command Line Arguments manually, re-check them - it's probably your own fault.
+
+> **Hint:** In your .minecraft folder, you'll find a file called `export.log` which contains information about FFmpeg's rendering process.
+
+If you did **not** customize the **Command Line Arguments**, you might have entered some insanely high (or low) values e.g. for Bitrate or Video Resolution. Try again with other, more reasonable values.
+
+## Compatibility with other Mods [compatibility]
+### General information [general]
+In General, the Replay Mod _should_ be compatible with most Forge and Fabric Mods.
+
+### Shaders Mod [shaders]
+[Karyonix' Shaders Mod](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1286604-shaders-mod-updated-by-karyonix)
+is no longer compatible with Minecraft Forge starting with 1.9.4. As such it is not compatible with the Replay Mod either.
+Please use [Optifine](https://optifine.net/) instead.
+
+### Custom Main Menu [custom-main-menu]
+The [Custom Main Menu](https://www.curseforge.com/minecraft/mc-mods/custom-main-menu) mod is often used in mod packs to customize their Main Menu with a button layout fitting the background image, links to their website / bug tracker and similar.
+If you are familiar with it, the button ids for the Replay Mod are: **17890234** (text: `replaymod.gui.replayviewer`) and **17890237** (text: `replaymod.gui.replayeditor`).
```

Comment:
The editor button has been removed from the main menu as well., There's now only a single button.